app/javascript/components/pages/Splash.js

```diff
@@ -1,12 +1,38 @@
-
 import React, { Component } from "react"
+import { NavLink } from 'react-router-dom'
+import PokeLogo from '../../../assets/images/PokeLogo1-1-removebg-preview.png'
+import {
+    Container,
+    Collapse,
+    Navbar,
+    NavbarToggler,
+    NavbarBrand,
+    Nav,
+    NavItem,
+  } from 'reactstrap';
+  import {
+    BrowserRouter as Router,
+    Route,
+    Switch
+  } from 'react-router-dom'
+
 
 
 class Splash extends Component{
 render() {
     return(
         <>
-            <h3>This is the Splash page </h3>
+        
+        <div 
+            className = "splash">
```

Comment:
```suggestion
        <div className = "splash">
```
indenting attributes work if you have many attributes/props for an element but if there's just one attribute the spacing gets wonky